api/src/main/java/org/openmrs/api/CohortService.java

```diff
@@ -134,6 +137,18 @@
 	
 	/**
 	 * Find all Cohorts that contain the given patient. (Not including voided Cohorts)
+	 *
+	 * @param patient
+	 * @return All non-voided Cohorts that contain the given patient
```

Comment:
If this returns non voided cohorts, then what is the use of the second boolean voided argument?